stdlib/Printf/docs/src/index.md

```diff
@@ -1,4 +1,4 @@
-# Printf
+# [Printf](@id man-printf)
```

Comment:
```suggestion
# Printf
```

This can already be referenced by name (e.g. `(Printf)[@ref]`)

Reply:
Doing so give me this error:
```
��� Warning: 'Printf' is not unique in src/base/io-network.md.
��� @ Documenter.CrossReferences /buildworker/worker/package_linux64/build/doc/deps/packages/Documenter/bFHi4/src/CrossReferences.jl:100
```

Reply:
There's a ["Printf" section on the NEWS page](https://docs.julialang.org/en/v1/NEWS/#Printf) as well, which is why it's not unique I believe.

Reply:
OK, the link is gone, but I leave the anchor, since it does no harm and may save someone figuring this out again some day.